src/ui/js/Game.js

```diff
@@ -1664,7 +1664,7 @@ Game.pageAddGameHeader = function(action_desc) {
       ') vs. ' + Api.game.opponent.playerName + ' (' +
       Api.game.opponent.button.name + ') ' + Game.SPACE_BULLET +
       'Round #' + Api.game.roundNumber;
-  $('title').html('Button Men Online &mdash; ' + gameTitle);
+  $('title').html(gameTitle + '&mdash; Button Men Online');
```

Comment:
This is whitespace but not a nit - you need whitespace inside the quotes before the `&mdash;` here, because as is it shows up as `Round #1- Button Men Online` for the player.